.eslintrc.json

```diff
@@ -85,6 +85,7 @@
       }
     ],
     "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-namespace": ["error", { "allowDeclarations": true }],
```

Comment:
Just for my own understanding. Why do we need this?

Reply:
So we can declare `hashRouting` in the `process.env` types in `src/index.ts`